### ISSUE TITLE
Element range attribute excludes the Binary type

### ISSUE THREAD

**Issue by hubblec4** [ietf-wg-cellar/ebml-specification#423]
Currently the `range` attribute is allowed only for numerical element types,
but Matroska has two Binary elements with a definde `range`.

https://github.com/ietf-wg-cellar/matroska-specification/issues/730

**Comment by robUx4** [comment]
Indeed, either we don't use the `range` field or we change the rules here. The text is pretty clear that it's only for "number" values: https://www.rfc-editor.org/rfc/rfc8794.html#expression-of-range

I think the idea of the `range` field, rather than just being rules written in the text, is that they are machine readable and parseable, to generate code accordingly. It becomes tricky to describe with that field for complex binary blobs, which EBML or Matroska are not supposed to interpret on their own.

So in the spirit of the separation of parsing levels, I would not put binary ranges inside the `range` field but in the text or in the `usage notes`.

**Comment by hubblec4** [comment]
> I would not put binary ranges inside the range field but in the text or in the usage notes.

OK. Sounds logical and I think `usage notes` could be enough for teh moment.

Would a new range-attribute makes sense, -> `range_binary` ?
